sdk/eventgrid/azure-messaging-eventgrid/src/main/java/com/azure/messaging/eventgrid/systemevents/AcssmsReceivedEventData.java

```diff
@@ -0,0 +1,65 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+// Code generated by Microsoft (R) AutoRest Code Generator.
+
+package com.azure.messaging.eventgrid.systemevents;
+
+import com.azure.core.annotation.Fluent;
+import com.fasterxml.jackson.annotation.JsonProperty;
+import java.time.OffsetDateTime;
+
+/** The AcssmsReceivedEventData model. */
+@Fluent
+public final class AcssmsReceivedEventData extends AcssmsEventBaseProperties {
```

Comment:
The word boundary was not correctly interpreted by AutoRest. This should have bee `AcsSmsReceivedEventData`. You can use directives in autorest to rename these models.